kafka-connect-influxdb/src/main/scala/com/datamountaineer/streamreactor/connect/influx/converters/InfluxPoint.scala

```diff
@@ -135,14 +144,14 @@ object InfluxPoint {
         )
     }
 
-  def writeField(builder: Point.Builder)(field: String, v: Any): Try[Point.Builder] = v match {
+  def writeField(builder: Point)(field: String, v: Any): Try[Point] = v match {
     case value: Long                 => Try(builder.addField(field, value))
-    case value: Int                  => Try(builder.addField(field, value))
+    case value: Int                  => Try(builder.addField(field, value.toLong))
     case value: BigInt               => Try(builder.addField(field, value))
-    case value: Byte                 => Try(builder.addField(field, value.toShort))
-    case value: Short                => Try(builder.addField(field, value))
+    case value: Byte                 => Try(builder.addField(field, value.toLong))
+    case value: Short                => Try(builder.addField(field, value.toLong))
     case value: Double               => Try(builder.addField(field, value))
-    case value: Float                => Try(builder.addField(field, value))
+    case value: Float                => Try(builder.addField(field, value.toDouble))
```

Comment:
it's this going to break compatibility when upgrading an existing connector?

Reply:
No, as there have been no changes to the influxdb types - only the mapping which is done invisibly by the Java library.  As this is an increase in precision there will be no impact except this explicit conversion in the connector code.